File: JavaScript/1048.js
var input = require('fs').readFileSync('dev/stdin', 'utf8');
var lines = input.split('\n');

var salario = parseFloat(lines.shift()); 

if (salario >= 0 && salario <= 400.00) {
  val = 0.15;
}
else if (salario >= 400.01 && salario <= 800.00) {
  val = 0.12;
}
else if (salario >= 800.01 && salario <= 1200.00) {
  val = 0.1;
}
else if (salario >= 1200.01 && salario <= 2000.00) {
  val = 0.07;
}
else {
  val = 0.04;
}

reaj = salario * val;
novosal = salario + reaj;
perc = val * 100;
perc = parseInt(perc)

console.log("Novo salario: " + novosal.toFixed(2));
console.log("Reajuste ganho: " + reaj.toFixed(2));
console.log("Em percentual: " + perc + " %")

File: Python/1048.py
salario = float(input())

if(salario >= 0 and salario <= 400):
	reaj = salario * 0.15
	novosal = salario + reaj
	perc = 15

elif(salario >= 400.01 and salario <= 800.00):
  reaj = salario * 0.12
  novosal = salario + reaj
  perc = 12

elif(salario >= 800.01 and salario <= 1200.00):
  reaj = salario * 0.1
  novosal = salario + reaj
  perc = 10

elif(salario >= 1200.01 and salario <= 2000.00):
	reaj = salario * 0.07
	novosal = salario + reaj
	perc = 7

elif(salario > 2000):
	reaj = salario * 0.04
	novosal = salario + reaj
	perc = 4

print(f"Novo salario: {novosal:.2f}")
print(f"Reajuste ganho: {reaj:.2f}")
print(f"Em percentual: {perc} %")